modules/agar/src/main/ts/ephox/agar/api/PhantomSkipper.ts

```diff
@@ -0,0 +1,16 @@
+import { before } from '@ephox/bedrock-client';
+
+const detect = (): boolean => navigator.userAgent.indexOf('PhantomJS') > -1;
+
+const bddSetup = (): void => {
+  if (detect()) {
+    before(function () {
```

Comment:
Is using `function` and `this` going to be necessary for all tests where we do things like this?

Reply:
Yes unfortunately, as that's what's required by the mocha implementation (they actually suggest avoiding arrow functions altogether: https://mochajs.org/#arrow-functions)